source/h2_stream.c

```diff
@@ -446,6 +450,20 @@ void aws_h2_stream_complete(struct aws_h2_stream *stream, int error_code) {
     s_h2_stream_destroy_pending_writes(stream);
 
     /* Invoke callback */
+    if (stream->base.on_metrics) {
+        if (stream->base.metrics.send_end_timestamp_ns == 0 && stream->base.metrics.send_start_timestamp_ns != 0) {
```

Comment:
Should be on `-1` now?

Reply:
Just an idea (not sure good or bad), we can also have a const variable with the default value so that we do not need to update it everywhere manually.